bin/dump_data.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Dump component data
+# Run this script from the root of the repository
+# dump_data.sh for all apps or dump_data.sh zaken documenten ... to specify specific apps
+# Note that postgres 17 requires postgres-client-17
+#
+# By default a schema and data dump are created separately. This can be changed with the flags --data-only, --schema-only
+# or --combined which appends the data dump to the schema dump.
+# The schema dump could not use -t to filter tables because this excludes extensions like postgis in the dump.
+# pg_dump also does not add related tables automatically, so `dump_data.sh zaken` does not add related zaaktype data to the dump.
+
+
+set -e
+
+DEFAULT_APPS=(besluiten catalogi documenten zaken)
```

Comment:
One thing that's important to note is that EnkelvoudigInformatieObjecten point to specific files, so we should at least mention in the docs that the same files must exist in the new Open Zaak instance storage to properly export documenten.